funfact/model/_factorization.py

```diff
@@ -44,15 +45,30 @@ def __init__(self, tsrex, initialize=True, nvec=1):
             tsrex = tsrex | self._leaf_initializer
         tsrex = tsrex | self._shape_analyzer | self._einspec_generator
         self._tsrex = tsrex
+        self._nvec = nvec
 
     @property
     def tsrex(self):
         return self._tsrex
 
+    @property
+    def nvec(self):
+        return self._nvec
+
     @property
     def shape(self):
         return self._tsrex.root.shape
 
+    @classmethod
```

Comment:
Why do we need to make it a `classmethod`?

Reply:
I was thinking of it as a constructor-like method that returns a new instance of the class, which is why this seemed appropriate.

Reply:
We can create an instance by `type(self)(tsrex, ...)`, so there is no need to make it a classmethod just for the purpose of obtaining `cls`. Besides, it could be tedious if users have to do
```
fac.devectorize(fac, i)
```
where `fac` is repeated twice.